src/pages/UpdateServers.js

```diff
@@ -0,0 +1,198 @@
+        this.state[list].filter(s => server.uid === s.uid).forEach(match => {
+          const updated_server = getMergedServer(match, server, MODEL_SERVER_PROPS);
+          putJson('/api/v1/server', JSON.stringify(updated_server));
+        });
+      }
+    }
+  }
+
+  assembleProcessPages = (theProps) => {
+    let pages = [];
+
+    pages.push({
+      name: 'PrepareReplace',
+      component: UpdateServerPages.PrepareReplace
+    });
+
+    //TODO other pages based on the theProps
+
+    if(theProps.installOS) {
+      pages.push({
+        name: "InstallOS",
+        component: UpdateServerPages.InstallOS
+      });
+    }
+
+    pages.push({
+      name: 'UpdateComplete',
+      component: UpdateServerPages.UpdateComplete
+    });
+    return pages;
+  }
+
+  replaceServer = (server, theProps) =>  {
+    let model =
+      updateServersInModel(server, this.state.model, MODEL_SERVER_PROPS_ALL, server.id);
+    this.props.updateGlobalState('model', model);
+
+    //console.log(JSON.stringify(model.getIn(['inputModel','servers'])));
+
+    if(theProps.selectedServerId) {
+      this.updateServerForReplaceServer(server);
+    }
+
+    //TODO only pass id and ip for now
+    theProps.server = {id : server.id, 'ip': server['ip-addr']};
+
+    let pages = this.assembleProcessPages(theProps);
+    //trigger update process to start
+    this.props.startUpdateProcess('ReplaceServer', pages, theProps);
+  }
+
+  renderCollapsibleTable() {
+    let tableConfig = {
+      columns: [
+        {name: 'id'},
+        {name: 'uid', hidden: true},
+        {name: 'ip-addr',},
+        {name: 'server-group'},
+        {name: 'nic-mapping'},
+        {name: 'mac-addr'},
+        {name: 'ilo-ip', hidden: true},
+        {name: 'ilo-user', hidden: true},
+        {name: 'ilo-password', hidden: true},
+        {name: 'role', hidden: true}
+      ]
+    };
+
+    return (
+      <CollapsibleTable
+        addExpandedGroup={this.addExpandedGroup} removeExpandedGroup={this.removeExpandedGroup}
+        model={this.state.model} tableConfig={tableConfig} expandedGroup={this.state.expandedGroup}
+        replaceServer={this.replaceServer} updateGlobalState={this.props.updateGlobalState}
+        autoServers={this.state.autoServers} manualServers={this.state.manualServers}/>
+    );
+  }
+
+  render() {
+    return (
+      <div className='wizard-page'>
+        <div className='content-header'>
+          <div className='titleBox'>
+            {this.renderHeading(translate('common.servers'))}
+          </div>
+          <div className='buttonBox'>
+            <div className='btn-row'>
+              <ActionButton type='default'
+                displayLabel={translate('collapse.all')} clickAction={() => this.collapseAll()} />
+              <ActionButton type='default'
+                displayLabel={translate('expand.all')} clickAction={() => this.expandAll()} />
+            </div>
+          </div>
+        </div>
+        <div className='wizard-content'>
```

Comment:
one thing about using the wizard css classes is that its causing an overly large header and clips the scrolling awkwardly...  theres a couple of solutions, one that I tried successfully was to add another css class here that sets the height back to auto... it might make more sense to just not have things that arent part of the wizard be under "wizard-page" and "wizard-content" though if that creates more css issues than it solves it might not be worth it

Reply:
hmm it does mess with the margins and whatnot to remove those :-/ will push a patchset with the css class approach